### ISSUE TITLE
Getting error while tagging image as latest.

### ISSUE THREAD

**Issue by PreetiCK** [wouterd/docker-maven-plugin#71]
Hi,

I have installed docker 1.6.  I am using this plugin to build, tag and push the image to dockerhub. But I am facing problem while tagging the image as latest. 
The same will work with Docker version 1.3. But I am getting the tag error with docker version 1.6. Please help me what I need to change in order to make it work.

Below is error stack trace
-------------------------------------------------------------------------------------------------------------------------------
[INFO] --- docker-maven-plugin:3.1-SNAPSHOT:tag-images (tag) @ VehicleBusiness ---
[INFO] Using docker provider: remote
[INFO] Tagging image '2bba9b26af01' with tag 'preetick/business:latest'..
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 27.498 s
[INFO] Finished at: 2015-06-29T12:43:46+00:00
[INFO] Final Memory: 15M/92M
[INFO] ------------------------------------------------------------------------
Waiting for Jenkins to finish collecting data
[ERROR] Failed to execute goal net.wouterdanes.docker:docker-maven-plugin:3.1-SNAPSHOT:tag-images (tag) on project VehicleBusiness: Failed to add tag '2bba9b26af01' to image 'preetick/business:latest': Conflict -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[JENKINS] Archiving /var/lib/jenkins/workspace/VehicleBusiness/pom.xml to com.mindtree.vehiclebusiness.entity/VehicleBusiness/3.2/VehicleBusiness-3.2.pom
[JENKINS] Archiving /var/lib/jenkins/workspace/VehicleBusiness/target/VehicleBusiness.war to com.mindtree.vehiclebusiness.entity/VehicleBusiness/3.2/VehicleBusiness-3.2.war
channel stopped
Finished: FAILURE

**Comment by wouterd** [comment]
Hi!
I'm pretty sure this is a bug that I fixed in an unrelated branch. I'll merge the bug fix asap. (somewhere tonight). It has to do with a change in docker where it doesn't allow retagging something without explicitly forcing it to.

tl;dr: you're doing nothing wrong. :)

**Comment by wouterd** [comment]
I just released v3.0.1 of the plugin, it should be available in maven central within a few hours. That should fix your issue. Let me know.

**Comment by PreetiCK** [comment]
Thank you. It worked :) . I forgot to acknowledge.

**Comment by wouterd** [comment]
Thanks for the acknowledgement, good to hear it's working. :)
